ingestion/src/metadata/clients/aws_client.py

```diff
@@ -144,15 +144,15 @@ def create_session(self) -> Session:
     def get_client(self, service_name: str) -> Any:
         # initialize the client depending on the AWSCredentials passed
         if self.config is not None:
-            logger.info(f"Getting AWS client for service [{service_name}]")
+            logger.debug(f"Getting AWS client for service [{service_name}]")
             session = self.create_session()
             if self.config.endPointURL is not None:
                 return session.client(
                     service_name=service_name, endpoint_url=str(self.config.endPointURL)
                 )
             return session.client(service_name=service_name)
 
-        logger.info(f"Getting AWS default client for service [{service_name}]")
+        logger.debug(f"Getting AWS default client for service [{service_name}]")
```

Comment:
since we're now passing the client to the source externally, this log was too chatty